bin/tictactoe

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+TicTacToe.new.call
```

Comment:
Exactly how the bin/tictactoe should look. Great encapsulation of CLI in separate class.